### DISCUSSION TITLE
Slurmweb.errors.SlurmwebAgentError: Unable to retrieve cluster name from agent

### DISCUSSION THREAD

**Discussion by sathwikreddy56** [rackslab/Slurm-web#384] [Q&A]
![image](https://github.com/user-attachments/assets/d25c3279-2172-432a-b4c2-aaa7ef2a2b4b)

**Comment by rezib** [comment]
Hello @sathwikreddy56 what is the Linux distribution you are using on this host? What is the version of `python3-requests` package? What version of Slurm are you using here?

**Reply by sathwikreddy56** [reply]
I have resloved the issue it was that the agent configurate agent.ini had a different version set which caused this issue
